M/manifold/build_tarballs.jl

```diff
@@ -0,0 +1,59 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "manifold"
+version = v"2.4.5"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/elalish/manifold.git", "6d932a2c7f3a269f6d280545487131c38d05f0ee"),
+    # DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/manifold
+mkdir build
+cd build/
+git submodule update --init --recursive
+cmake .. \
+    -DCMAKE_INSTALL_PREFIX=$prefix \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DMANIFOLD_PAR=NONE \
+    -DMANIFOLD_TEST=OFF \
+    -DBUILD_TEST_CGAL=OFF \
+    -DMANIFOLD_PYBIND=OFF \
+    -DMANIFOLD_JSBIND=OFF \
+    -DMANIFOLD_EXPORT=OFF \
+    -DBUILD_SHARED_LIBS=ON
+cmake --build . --parallel ${nproc}
+cmake --install .
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+platforms = filter(platforms) do p
+    !Sys.isfreebsd(p) && libc(p) != "musl"
+end
+platforms = expand_cxxstring_abis(platforms)
+
+# The products that we will ensure are always built
+products = [
+    # LibraryProduct("libClipper2", :libClipper2),
+    # LibraryProduct("libmanifold", :libmanifold),
+    # LibraryProduct("libglm", :libglm),
+    LibraryProduct("libmanifoldc", :libmanifoldc)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = Dependency[
+]
```

Comment:
```suggestion
    Dependency("Clipper2_jll")
]
```
I just saw that this is building its own Clipper2 and shipping it, and since the 2.4.0 version they have the ability to use a system-provided Clipper2 library. Let's try using that to reduce duplication of libraries.

(As an aside, we probably need to update Clipper2 to be 1.3.0 now, since we are at 1.2.2. @ferrolho would this update be a problem to your work on Clipper2 in JuliaGeometry?)

Reply:
No, this is not a problem at all. I spent some time trying to use Clipper2 from Julia, but I never got around to finishing things.